zyte_common_items/pages.py

```diff
@@ -92,6 +97,58 @@ async def currencyRaw(self) -> Optional[str]:
         return None
 
 
+class DescriptionMixin(FieldsMixin):
+    """Provides description and descriptionHtml field implementations."""
+
+    _description_html: Union[HtmlElement, str, None] = None
+    _description_str: Optional[str] = None
+
+    _description_default = False
+    _descriptionHtml_default = False
```

Comment:
these flags are just for avoiding recursive calls of default fields, I haven't found a cleaner check